node-fs-writefile/cp.js

```diff
@@ -0,0 +1,8 @@
+const fs = require('fs');
+const writeFile = process.argv[3];
+fs.readFile(`${process.argv[2]}`, 'utf-8', (err, data) => {
+  if (err) throw err;
+  fs.writeFile(`${writeFile}`, data + '\n', 'utf-8', err => {
```

Comment:
For making file copies, it'd be best to avoid inserting any new characters.

Reply:
Oh okay, what do you mean by 'new characters'.